### ISSUE TITLE
[Bug]: Anonymous blocks can not be redebugged

### ISSUE THREAD

**Issue by runzi389205** [oceanbase/odc#124]
### ODC version

4.2.1

### OB version

4.1.0

### What happened?

while anonymous blocks redebugged，
![image](https://github.com/oceanbase/odc/assets/141202865/b0577251-1dc4-495e-8c8a-f2f11c30a00a)

Error：BadRequest exception, type=NullPointerException, message=AnonymousBlock can not be null for StartPLDebugReq

response info：
```json
{
    "code": "BadRequest",
    "durationMillis": 27,
    "error": {
        "code": "BadRequest",
        "details": [
            {
                "code": "BadRequest",
                "message": "BadRequest exception, type=NullPointerException, message=AnonymousBlock can not be null for StartPLDebugReq",
                "target": null
            },
            {
                "code": "NullPointerException",
                "message": "AnonymousBlock can not be null for StartPLDebugReq",
                "target": null
            }
        ],
        "message": "错误的请求，详情：BadRequest exception, type=NullPointerException, message=AnonymousBlock can not be null for StartPLDebugReq"
    },
    "httpStatus": "BAD_REQUEST",
    "message": "错误的请求，详情：BadRequest exception, type=NullPointerException, message=AnonymousBlock can not be null for StartPLDebugReq",
    "requestId": "ITJPP3PSTE8HWEULR68S4V",
    "server": "sqaappecsv62s2011162217057.sa128",
    "successful": false,
    "timestamp": 1693793366.954000000,
    "traceId": "2ff5a9d4ca1b4304"
}
```
### What did you expect to happen?

re-enter the pl debug state

### Cloud

**Comment by smallsheeeep** [comment]
The parameter is missing `anonymousBlock` field, so the issue may should be fixed by frontend. 
<img width="1401" alt="image" src="https://github.com/oceanbase/odc/assets/84772536/224771e6-159b-4530-8d9b-5f5f1ec79224">
<img width="1404" alt="image" src="https://github.com/oceanbase/odc/assets/84772536/5b15890a-3120-46cb-b524-0d7e972e6de4">

**Comment by HSunboy** [comment]
Fixed In Commit [cbdee53be86e7702447bb38c18ae3066308ea4c4](https://github.com/oceanbase/odc-client/commit/cbdee53be86e7702447bb38c18ae3066308ea4c4)
            ------
            晓康
            Wed Sep 6 15:46:29 2023 +0800

**Comment by runzi389205** [comment]
PASS


![Image](https://github.com/oceanbase/odc/assets/141202865/09934b5b-ff09-4aac-8099-85eeefaf368e)



![Image](https://github.com/oceanbase/odc/assets/141202865/2c2c1674-c3c3-4687-9144-4d4537c95789)
